grakn-graph/src/test/java/ai/grakn/graph/internal/CastingTest.java

```diff
@@ -72,40 +69,6 @@ public void hashCodeTest() throws Exception {
     }
 
     @Test
-    public void testGetRole() throws Exception {
-        assertEquals(role, casting.getRole());
-        Vertex vertex = graknGraph.getTinkerPopGraph().addVertex(Schema.BaseType.CASTING.name());
-        vertex.property(Schema.ConceptProperty.ID.name(), vertex.id().toString());
-
-        CastingImpl casting2 = graknGraph.getConcept(ConceptId.of(vertex.id().toString()));
-        boolean exceptionThrown = false;
-        try{
-            casting2.getRole();
-        } catch(NoEdgeException e){
-            exceptionThrown = true;
-        }
-        assertTrue(exceptionThrown);
-
-
-        TypeImpl c1 = (TypeImpl) graknGraph.putEntityType("c1'");
-        TypeImpl c2 = (TypeImpl) graknGraph.putEntityType("c2");
-        Vertex casting2_Vertex = graknGraph.getTinkerPopGraph().traversal().V(casting2.getBaseIdentifier()).next();
-        Vertex c1_Vertex = graknGraph.getTinkerPopGraph().traversal().V(c1.getBaseIdentifier()).next();
-        Vertex c2_Vertex = graknGraph.getTinkerPopGraph().traversal().V(c2.getBaseIdentifier()).next();
-
-        casting2_Vertex.addEdge(Schema.EdgeLabel.ISA.getLabel(), c1_Vertex);
-        casting2_Vertex.addEdge(Schema.EdgeLabel.ISA.getLabel(), c2_Vertex);
-
-        exceptionThrown = false;
-        try{
-            casting2.getRole();
-        } catch(MoreThanOneEdgeException e){
-            exceptionThrown = true;
-        }
-        assertTrue(exceptionThrown);
-    }
```

Comment:
No longer valid. If you look through the test carefully you will realise that such a situation is not possible without severally hacking the API. For example this test changes the type of concepts. Which has not been possible for a long time.